projects/meterBridge/index.js

```diff
@@ -13,7 +15,15 @@ function getTvl(chain) {
             [json.Wrapped]:
                 (await sdk.api.eth.getBalance({ target, chain: chain })).output
         };
-        // console.log("balances:",balances,chain)
+        if (chain == "ethereum") {
```

Comment:
the code here is bit outdated, I have fixed it here: https://github.com/DefiLlama/DefiLlama-Adapters/commit/d8deb30a8cfbfcf48c82f34349d61643fafe9501